File: Lab4_Romanov/Task 2/hw7(2).py
def main():
    print("Элементы множества A")
    A = set(map(int, input().split()))

    print("Элементы множества B")
    B = set(map(int, input().split()))

    in_only_A = A - B

    return f"Элементы которые есть только в A: {in_only_A}"
print(main())

File: Lab4_Romanov/Task 2/hw7(2).cpp
#include <iostream>
#include <set>

using namespace std;

int main() {
    setlocale(LC_ALL, "russian");
    cout << "Элементы множества A через пробел:" << endl;

    set<int> A;
    int num;

    while (cin >> num) {
        A.insert(num);
        if (cin.peek() == '\n') break;
    }
    cin.clear();

    cout << "Элементы множества B через пробел:" << endl;

    set<int> B;

    while (cin >> num) {
        B.insert(num);
        if (cin.peek() == '\n') break;
    }

    set<int> in_only_A;
    for (int element : A) {
        if (B.find(element) == B.end()) {
            in_only_A.insert(element);
        }
    }

    cout << "Элементы которые есть только в A: ";
    for (int element : in_only_A) {
        cout << element << " ";
    }
    cout << endl;

    return 0;
}
